src/main/java/com/github/dockerjava/api/model/TaskDefaults.java

```diff
@@ -0,0 +1,61 @@
+package com.github.dockerjava.api.model;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.dockerjava.core.RemoteApiVersion;
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.apache.commons.lang.builder.ToStringBuilder;
+import org.apache.commons.lang.builder.ToStringStyle;
+
+import javax.annotation.CheckForNull;
+import java.io.Serializable;
+
+/**
+ * @since {@link RemoteApiVersion#VERSION_1_24}
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class TaskDefaults implements Serializable {
+
+    public static final Long serialVersionUID = 1L;
+
+    /**
+     * @since 1.24
+     */
+    @JsonProperty("LogDriver")
+    private Driver logDriver;
```

Comment:
It's called `Driver` in the `engine-api` too, and it's pretty generic, so might be the case that it is used for other drivers in the future.